src/scratchtocatrobat/tools/svgtopng.py

```diff
@@ -210,6 +210,14 @@ def _parse_and_rewrite_svg_file(svg_input_path, svg_output_path):
     if 'width' in root.attrib and float((root.attrib['width']).strip('px%')) <= 0:
         root.attrib['width'] = '1'
 
+    #some SVGs do not contain height and width attributes - viewbox attribute can be used to determine those parameters
+    if not 'height' in root.attrib and not 'width' in root.attrib:
```

Comment:
what if one of both is missing? (only hight or width present)
would an "OR" make more sense than "and"?

Reply:
So far I have not found a case like this, but an OR would maybe be safer. I will check this and probably change it. Also I just found out I made a mistake involving the first two entries of the viewBox.